pkg/kubectl/resource_printer.go

```diff
@@ -322,13 +322,20 @@ type handlerEntry struct {
 type PrintOptions struct {
 	NoHeaders          bool
 	WithNamespace      bool
+	WithKind           bool
 	Wide               bool
 	ShowAll            bool
 	ShowLabels         bool
 	AbsoluteTimestamps bool
+	KindName           string
 	ColumnLabels       []string
 }
 
+type PrintKindOptions struct {
```

Comment:
`PrintOptions` is not instantiated publicly in `HumanReadablePrinter` (https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/resource_printer.go#L346). 

To set `WithKind` and `KindName` from [this line](https://github.com/juanvallejo/kubernetes/blob/e10a59f93504c87367bf2f052af3639f73aa8b63/pkg/kubectl/cmd/get.go#L345), I instantiate the new struct `PrintKindOptions` as a public field [here](https://github.com/juanvallejo/kubernetes/blob/e10a59f93504c87367bf2f052af3639f73aa8b63/pkg/kubectl/resource_printer.go#L367).